packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -255,21 +277,24 @@ export default {
         }
     },
 
-    async mounted () {
-        if (this.isAuthFinished) {
-            await this.initialise();
-        }
-    },
-
     methods: {
         ...mapActions('fAccountInfoModule', [
             'loadConsumerDetails',
             'editConsumerDetails'
         ]),
 
+        /**
+        * Informs the template that we are in an Error State.
+        * @param {object} Error - The error that has recently occurred
+        */
+        handleErrorState (error) {
+            this.shouldShowErrorPage = true;
+            this.error = error;
+        },
+
         /**
         * Gets the form data (from the api) and assigns it to State
-        * then lowers the isFormDirty flag as the form data is currently clean
+        * then lowers the hasFormUpdate flag as the form data is currently clean
         * then stops the on-screen spinner from showing
```

Comment:
@oliversweb I noticed account-info is using `hasFormUpdate` and contact-preferences uses `isFormDirty` shall I update use `isFormDirty` so we are consistent?

Reply:
I think @kevinrodrigues opted for the `hasFormUpdate` because the is dirty term could be confused with the vue validate solution which also uses that term. So I suppose who ever returns back to the f-contact-preferences should change the variable name there to match f-account-info 👍